packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,105 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const ignoredOperations = [
+  /* missing route */
+  /* #5694 */
+  "getSyncCommitteeRewards",
+  "getBlockRewards",
+  "getAttestationsRewards",
+  "getDepositSnapshot", // Won't fix for now, see https://github.com/ChainSafe/lodestar/issues/5697
+  "getBlindedBlock", // #5699
+  "getNextWithdrawals", // #5696
+  "getDebugForkChoice", // #5700
+  /* #6080 */
+  "getLightClientBootstrap",
+  "getLightClientUpdatesByRange",
+  "getLightClientFinalityUpdate",
+  "getLightClientOptimisticUpdate",
+  "getPoolBLSToExecutionChanges",
+  "submitPoolBLSToExecutionChange",
+];
+
+const ignoredProperties: Record<string, IgnoredProperty> = {
+  /*
+   #5693
+   missing finalized
+   */
+  getStateRoot: {responseProperties: ["finalized"]},
+  getStateFork: {responseProperties: ["finalized"]},
+  getStateFinalityCheckpoints: {responseProperties: ["finalized"]},
+  getStateValidators: {responseProperties: ["finalized"]},
+  getStateValidator: {responseProperties: ["finalized"]},
+  getStateValidatorBalances: {responseProperties: ["finalized"]},
+  getEpochCommittees: {responseProperties: ["finalized"]},
+  getEpochSyncCommittees: {responseProperties: ["finalized"]},
+  getStateRandao: {responseProperties: ["finalized"]},
+  getBlockHeaders: {responseProperties: ["finalized"]},
+  getBlockHeader: {responseProperties: ["finalized"]},
+  getBlockV2: {responseProperties: ["finalized"]},
+  getBlockRoot: {responseProperties: ["finalized"]},
+  getBlockAttestations: {responseProperties: ["finalized"]},
+  getStateV2: {responseProperties: ["finalized"]},
+
+  /* 
+   #6168
+   /query/syncing_status - must be integer
+   */
+  getHealth: {requestProperties: ["query.syncing_status"]},
+
+  /**
+   * #6185
+   *  - must have required property 'query'
+   */
+  getBlobSidecars: {requestProperties: ["query"]},
+
+  /* 
+   #4638 
+   /query - must have required property 'skip_randao_verification'
+   */
+  produceBlockV2: {requestProperties: ["query.skip_randao_verification"]},
+  produceBlindedBlock: {requestProperties: ["query.skip_randao_verification"]},
+};
+
 const openApiJson = await fetchOpenApiSpec(openApiFile);
-runTestCheckAgainstSpec(openApiJson, routesData, reqSerializers, returnTypes, testDatas, {
-  // TODO: Investigate why schema validation fails otherwise
-  routesDropOneOf: ["produceBlockV2", "produceBlindedBlock", "publishBlindedBlock"],
-});
+runTestCheckAgainstSpec(
+  openApiJson,
+  routesData,
+  reqSerializers,
+  returnTypes,
+  testDatas,
+  {
+    // TODO: Investigate why schema validation fails otherwise
```

Comment:
Might be good ref the issue you created for this (to get more context)
- https://github.com/ChainSafe/lodestar/issues/6187